pa/fin/serializers/ticker.py

```diff
@@ -157,6 +158,27 @@ def get_debt(self, obj):
             }
         return None
 
+    def get_shares_dilution(self, obj):
+        """
+        Return shares dilution rate
+        """
+        current_shares_amount = obj.ticker_statements \
```

Comment:
If I correct - PEP recommend to avoid backslashes 

it can be changed to: 
```python 
current_shares_amount = (
    obj.ticker_statements 
            .filter(name=Statements.outstanding_shares) 
            .order_by('-fiscal_date_ending') 
            .first()
)
```